modules/kafka/kafka-dest-worker.c

```diff
@@ -296,6 +296,38 @@ kafka_dest_worker_transactional_insert(LogThreadedDestWorker *s, LogMessage *msg
   if (result != LTR_SUCCESS)
     return result;
 
+  return LTR_SUCCESS;
```

Comment:
`_transaction_commit()` is missing from `kafka_dest_worker_transactional_insert`